src/pcre2_config.c

```diff
@@ -237,6 +242,7 @@ switch (what)
     return (int)(1 + ((where == NULL)?
       strlen(v) : PRIV(strcpy_c8)((PCRE2_UCHAR *)where, v)));
     }
+  break;
```

Comment:
Is this `break` a line of dead code? I can't see how it's reached, given that there's a `return` immediately above.

Reply:
Yes, but added for consistency and symmetry with the other `case` around it (see PCRE2_CONFIG_UNICODE_VERSION above).

Again, I have no strong feeling eitherway, but would rather have them all be consistent, specially when some have already the dead code and you didn't seem to argue about it one way or the other even when it was recently changed (see alignment fix above) in line [208](https://github.com/PCRE2Project/pcre2/blob/be43aa76e766409cf296abae0b331a0de1ccfc7b/src/pcre2_config.c#L208)

Reply:
Oh! I'm sorry if I'm being inconsistent. My apologies.

It's a tiny nit - I don't have a very strong feeling either.

My personal preference though is that I don't like lines of unreachable dead code after a return.

Reply:
I also personally prefer not to have dead code, but it can be useful sometimes in this codebase that has very large code segments and an unusual indentation as a failsafe.

I had to admit I was motivated to add it when I had to stare for too long at:

https://github.com/PCRE2Project/pcre2/blob/507bdd0feca36a811a51a0056c88cea739dddb47/src/pcre2_config.c#L147-L156

But will remove this and all others in the file instead.